studies/templates/studies/study_responses.html

```diff
@@ -299,7 +307,9 @@ <h5 class="card-title">Response details</h5>
                                     {% for pair in response.summary %}
                                         <tr class="{% if pair.name == 'Video privacy level' %} {% if pair.value == 'public' %} list-group-item-success {% elif pair.value == 'scientific' %} list-group-item-warning {% else %} list-group-item-danger {% endif %} {% endif %}">
                                             <td tabindex="0" title="{{ pair.description }}">{{ pair.name }}</td>
-                                            <td>{{ pair.value }}</td>
+                                            {# djlint:off #}
+                                        <td>{{ pair.value }}{% if pair.name == 'Date created' %} UTC{% endif %}</td>
+                                            {# djlint:on #}
```

Comment:
I also turned off djlint here because it kept putting the `pair.value` and `if` template tags on separate lines, which introduced lots of empty space in all of the `td` elements, which in turn broke the string manipulation used to grab the parent name/IDs from this table.

Reply:
I'll remove djlint.